src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/AS_InbreedingCoeffUnitTest.java

```diff
@@ -0,0 +1,218 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
```

Comment:
that's why i think it would be better not to have a special package for allele specific stuff